### PR TITLE
Fabo/docker testnet

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,152 +1,146 @@
-{
```
Comment (jbibla) [inline package.json]: are we having linting issues? should we bring back prettier? 😞

Comment (faboweb) [reply package.json]: Hahahaha yes probably

File: .circleci/config.yml [hunk 2]
```diff
@@ -132,29 +132,6 @@ jobs:
             - app/dist
             - app/dist_testnet
 
-  # Build Gaia from the SDK source code specified by tasks/build/Gaia/VERSION
-  buildGaia:
```
Comment (jbibla) [inline .circleci/config.yml]: 🎉

File: package.json [hunk 3]
```diff
@@ -15,13 +15,9 @@
   },
   "scripts": {
     "build": "yarn build:keybase-cache && yarn build:ui",
-    "build:testnets": "cd tasks/build/testnets && ./localBuild.sh",
-    "build:gaia": "cd tasks/build/Gaia && ./localBuild.sh",
-    "build:local": "node tasks/build/local/build",
     "build:keybase-cache": "babel-node --presets @babel/env tasks/keybase-fallback.js",
     "build:ui": "cross-env NODE_ENV=production webpack --colors --config webpack.renderer.config.js --mode=production",
-    "start": "concurrently -k \"yarn frontend\" \"yarn backend\"",
-    "start:new": "concurrently -k \"yarn frontend\" \"yarn backend:new\"",
```
Comment (jbibla) [inline package.json]: no more `start:new`?

File: test/unit/specs/store/json/schema.graphql [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Welcome to AWS AppSync!
+#
+# Use this editor to define the structure of your API with the GraphQL
```
Comment (jbibla) [inline test/unit/specs/store/json/schema.graphql]: wow! what are we using this for?

File: test/unit/specs/store/json/x.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
+#**
+Make an arbitrary HTTP call when this field is listed in a query selection set.
+The "relativePath" is the resource path relative to the root URL provided when you
+created the HTTP data source. Pass a "params" object to forward headers, body,
+and query parameters to the HTTP endpoint.
+*#
+{
+  "version": "2018-05-29",
+  "method": "GET",
+  ## E.G. if full path is https: //api.xxxxxxxxx.com/posts then resourcePath would be /posts ** "resourcePath": "/staking/validators",
```
Comment (jbibla) [inline test/unit/specs/store/json/x.json]: invalid comment, maybe?

File: package.json (hunk 3 above)
Comment (faboweb) [reply package.json]: it's always new now, but the account stays always the same

File: test/unit/specs/store/json/schema.graphql (hunk 4 above)
Comment (faboweb) [reply test/unit/specs/store/json/schema.graphql]: removed ;)

File: test/unit/specs/store/json/x.json (hunk 5 above)
Comment (faboweb) [reply test/unit/specs/store/json/x.json]: removed ;) thx for finding those

**Pull request merged**